scripts/make_docs.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 dest=generated/static/docs
-"$(yarn bin)/typedoc" --theme src/static/typedoc/ --out $dest src --ignoreCompilerErrors
+"$(yarn bin)/typedoc" --theme src/static/typedoc/ --exclude "src/**/?(test_utils|*.test).ts"  --out $dest src --ignoreCompilerErrors
```

Comment:
This exclusion was needed because test files do not typecheck with the main webext tsconfig, is it okay not to run typedoc on these files?

Reply:
Yes, that's fine - we only use typedoc for user-facing documentation :)